nixos/doc/manual/configuration/xfce.xml

```diff
@@ -30,16 +30,6 @@
   (system wide), put them into your
   <xref linkend="opt-environment.systemPackages"/>.
  </para>
- <simplesect xml:id="sec-xfce-thunar-volumes">
-  <title>Thunar Volume Support</title>
```

Comment:
Maybe this was meant as a proxy for GVFS, when Thunar is used outside of Xfce?

Reply:
But not sure if we need that.

Reply:
It doesn't even seem to make sense to me at all either, it says to just enable xfce.